test-specifications/crypto-service/accountUpdateTransaction.md

```diff
@@ -187,14 +187,12 @@ The tests contained in this specification will assume that a valid account was a
-| 6       | Updates the expiration time of an account to the maximum period plus one second (8,000,002 seconds) from the current time             | accountId=<CREATED_ACCOUNT_ID>, expirationTime=<CURRENT_TIME> + 8000002, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update fails with an INVALID_EXPIRATION_TIME response code from the network.                              | N                 |
+| Test no | Name                                                                                        | Input                                                                                                                                       | Expected response                                                                                                     | Implemented (Y/N) |
+|---------|---------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Updates the expiration time of an account to 8,000,001 seconds from the current time        | accountId=<CREATED_ACCOUNT_ID>, expirationTime=<CURRENT_TIME> + 5184000, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update succeeds and the account's expiration time should equal 5,184,000 seconds from the current time.   | Y                 |
+| 2       | Updates the expiration time of an account to -1 seconds from the current time               | accountId=<CREATED_ACCOUNT_ID>, expirationTime=<CURRENT_TIME> -1, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>]        | The account update fails with an INVALID_EXPIRATION_TIME response code from the network.                              | Y                 |
+| 3       | Updates the expiration time of an account to 1 second less than its current expiration time | accountId=<CREATED_ACCOUNT_ID>, expirationTime=<CURRENT_TIME> + 2592000, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update succeeds and the account's auto renew period should equal 2,592,000 seconds from the current time. | Y                 |
+| 4       | Updates the expiration time of an account to 8,000,002 seconds from the current time        | accountId=<CREATED_ACCOUNT_ID>, expirationTime=<CURRENT_TIME> + 2591999, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update fails with an INVALID_EXPIRATION_TIME response code from the network.                              | N                 |
```

Comment:
I think there is a same kind of type as in test №1 above

Reply:
Fixed